redis/src/main/scala/zio/redis/options/Strings.scala

```diff
@@ -2,6 +2,25 @@ package zio.redis.options
 
 trait Strings {
 
+  sealed trait StralgoCommand
+
+  object StralgoCommand {
+    sealed case class StralgoLCS(`type`: StralgoType) extends StralgoCommand
+  }
+
+  sealed trait StralgoType { self =>
+    private[redis] final def stringify: String =
+      self match {
+        case StralgoType.Strings => "STRINGS"
+        case StralgoType.Keys    => "KEYS"
```

Comment:
I'm just looking around. Some options don't seem to be supported. List them? `[LEN] [IDX] [MINMATCHLEN <len>] [WITHMATCHLEN]
`

Reply:
true, some of these would also alter the output, that's why I'd like some feedback on the input command structure before delving deeper into the other options.